docs/apis/plugintypes/tinymce/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: TinyMCE Editor plugins
+title: Legacy TinyMCE Editor plugins
```

Comment:
As we've left only a single reference to "TinyMCE Editor plugins" in docs/apis/plugintypes, I would suggest moving this Legacy to a subfolder (having "TinyMCE Editor Plugins" as parent).